samples/bring-your-own-container/README.md

```diff
@@ -0,0 +1,32 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates the ability for customers to start training jobs using their own training scripts, packaged in a SageMaker-compatible container, using the Amazon SageMaker Operator for Kubernetes. 
```

Comment:
* Should we use `customers`.. I feel a user would more expect to read `user` here than the more Amazon-jargony `customer`.
* This first paragraph refer to users in third person ("for _customers_ to start ... using _their_ ..."), other parts of this doc refer to them as first-person ("This sample assumes that _you_ ..."). Can we make uniform?